tests/lib/snaps/test-snapd-policy-app-consumer/meta/snap.yaml

```diff
@@ -322,3 +325,5 @@ plugs:
     interface: dbus
     bus: system
     name: test.system
+  test-snapd:
+    interface: test-snapd
```

Comment:
Hmm, we have a new interface called `test-snapd`?

Reply:
Renaming to "dummy" as discussed on IRC (and I'm open to rename it again if we find a better name; also we might consider hiding the interface from the snap interfaces list if needed, in a followup PR potentially).

Reply:
Thanks, this sounds great